_data/pages/es/downloads.yml

```diff
@@ -12,6 +17,9 @@ images:
   tar: "/assets/media/icons/tar.png"
   jdbc: "/assets/media/icons/javajdbc.png"
   perftop: "/assets/media/icons/consol.png"
+  windows: "/assets/media/icons/windows.png"
+  zip: "/assets/media/icons/zip.png"
+  odbc: "/assets/media/icons/odbc.png"
```

Comment:
I'd resize this image in the source or in the CSS, but it's really large on screen. ~60-70% of its current size seems appropriate.

Reply:
I have resize the image now.